### PR TITLE
Tidying up spelling and grammar in README

### PR DESCRIPTION
**Pull request by freelance-tech-writer**
This loader has been quite helpful to me on a professional project, so I thought I'd help out with some copy edits. There are a few spelling and grammar fixes, as well as some small changes for consistency. Hope this helps!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,37 +63,37 @@ module.exports = {
...
+  * `[path]` - the path of the image
+  * `[hash]` - the hash or the image content
+  * `[pathhash]` - the hash or the image path
+* `angularBaseWorkaround` adds a workaround for issues with combining `<base>` and the history API (which is [typical for Angular.js](https://github.com/angular/angular.js/issues/8934)). Default is `false`.
 * `prefixize` isolates an image content by prefixing its `id`, `xlink:href` and `url(#id)` elements. Default is `true`.
-* `spriteModule` defines [custom sprite implementation](#custom-sprite-implementation) module path.
-* `esModule` whether to transpile module to ES compatible format. When this option is set to `true`, loader will produce `module.exports.__esModule = true; module.exports['default'] = svg`. Default is `false`. Useful for transpilers other than Babel.
+* `spriteModule` defines [custom sprite implementation](#custom-sprite-implementation) module path
+* `esModule` configures whether to transpile the module to an ES-compatible format. When this option is set to `true`, the loader will produce `module.exports.__esModule = true; module.exports['default'] = svg`. Default is `false`. (This is useful for transpilers other than Babel.)
```
Comment (rayshade) [inline README.md]: This is a good edit, because it adds something.

Comment (rayshade) [inline README.md]: Other edits don't change much, keep'em, or ignore'em as you like.

**Comment by kisenka** [comment]
@freelance-tech-writer thanks a lot for your contribution 👍

**Comment by princed** [comment]
@freelance-tech-writer Thank you so much! Could you be interested in fixing #30 maybe?

**Pull request merged**